.circleci/config.yml

```diff
@@ -201,6 +222,10 @@ workflows:
       - unit_test:
           name: unit_test_search_box
           <<: *requires_build_final
+
+      - unit_test_interaction:
+          name: unit_test_interaction_search_box
+          <<: *requires_build_final
       - unit_test_final:
```

Comment:
Add a blank line above to separate interaction tests from the final collector. That, or remove the blank line separating the unit tests from the interaction tests.

Reply:
Updated.